common/src/View/Grid.cpp

```diff
@@ -236,7 +240,7 @@ namespace TrenchBroom {
             }
             
             Vec3 normDelta = face->boundary().normal * dist;
-            size_t gridSkip = static_cast<size_t>(normDelta.dot(normDelta.firstAxis())) / actualSize();
+            size_t gridSkip = static_cast<size_t>(static_cast<size_t>(normDelta.dot(normDelta.firstAxis())) / actualSize());
```

Comment:
This line I was worried about. I'm not sure if it will work as intended when actualSize() is between 0 and 1.

Reply:
IIRC, `gridSkip`indicates the number of grid planes we have to skip further down in order to actually snap to the next grid plane after the face has travelled the intended distance given by `dist`. For the case of an actual size > 1, your change will not have any effect, as its essentially just rounding down to the next integer. For the case of an actual size < 1, the `gridSkip` value will increase which seems to be correct as there are more grid planes to skip. But I'm not sure either.

In the latter case (actual size < 1), the static cast you have added will lead to fewer grid planes skipped, right? So the drag will snap earlier, but I'm not sure if that's correct. Sorry, I can't really say and would ask you to add a test  if you can.

Reply:
> In the latter case (actual size < 1), the static cast you have added will lead to fewer grid planes skipped, right?

In general that's right, but for actual sizes < 1 we support (0.5, 0.25, 0.125), dividing by them will give exactly an integer, so the static cast doesn't reduce the result.